manifests/install.pp

```diff
@@ -1,7 +1,7 @@
 #
 # @api private
 #
-class ipmi::install {
+class ipmi::install inherits ipmi {
```

Comment:
Inheriting the base class shouldn't be necessary.

Reply:
Without this inheritance, the unit tests for the `define` fail as the toplevel class vars aren't instantiated.

Reply:
This works for me: 

```puppet
diff --git a/manifests/install.pp b/manifests/install.pp
index 88db7fa..5980fec 100644
--- a/manifests/install.pp
+++ b/manifests/install.pp
@@ -1,8 +1,10 @@
 #
 # @api private
 #
-class ipmi::install inherits ipmi {
+class ipmi::install {
   assert_private()
 
+  include ipmi
+
   ensure_packages($ipmi::packages)
 }
```

Reply:
When I do that, I get a dep loop on the top level create resources for `ipmi::users`.

Reply:
Did you use `include` or `require` for you test?

Reply:
Add, I did `require` that would probably do it.

Reply:
Fixed.